eidpdata/src/main/java/uk/ac/ucl/eidp/data/DBMapping.java

```diff
@@ -50,8 +50,9 @@ public void loadMapping() {
 
     public List<Map<String, String>> dbAction(String methodPath, Map<String, String> parameters) {
         
-        if (!methodPath.matches("[\\w-]*\\.[\\w-]*\\.[\\w-]*")) 
+        if (!methodPath.matches("[\\w-]*\\.[\\w-]*\\.[\\w-]*")) {
```

Comment:
\[Checkstyle\] WARNING: 'if' have incorrect indentation level 8, expected level should be 4\.